downstream/modules/platform/ref-sso-variables.adoc

```diff
@@ -0,0 +1,25 @@
+[id="ref-sso-variables"]
+
+= {RHSSO} variables
+
+*Use these variables for `automationhub` or `automationcatalog`.
+
+[cols="50%,50%",options="header"]
+|====
+| *Variable* | *Description* 
+| *`*_sso_console_admin_password`* | SSO admin password.
```

Comment:
The asterisk before the variable name could be confusing: users might think that you should prepend the variables with  automationhub or automationcatalog.